implants/lib/eldritch/Cargo.toml

```diff
@@ -45,6 +46,7 @@ tempfile = { workspace = true }
 tera = { workspace = true }
 tokio = { workspace = true , features = ["macros", "rt-multi-thread"] }
 tokio-stream = { workspace = true }
+uuid = { workspace = true, features = ["v4"] }
```

Comment:
If this is just used for testing could be moved to `dev-deps` not sure if it's used other places in eldritch.